doc/api/http2.md

```diff
@@ -1727,10 +1728,15 @@ changes:
 Returns a `net.Server` instance that creates and manages `Http2Session`
 instances.
 
+Since no browsers support [unencrypted HTTP/2][HTTP/2 Unencrypted],
+the use of [`http2.createSecureServer()`][] is preferred.
```

Comment:
Again,

```
Since there are no browsers known that support [unencrypted HTTP/2](HTTP/2 Unencrypted),
the use of [`http2.createSecureServer()`][] is necessary when communicating with browser
clients.
```

Reply:
Okie dokie 👍